packages/simulator/lib/region.js

```diff
@@ -189,7 +189,11 @@ class Region {
             mergeAll(),
             mergeMap(({ taxi, bookings }) =>
               from(bookings).pipe(
-                mergeMap((booking) => taxi.fleet.handleBooking(booking, taxi))
+                // TODO: We have a bug here, the system tries to dispatch taxis that are already full.
```

Comment:
@irony jag får ett från vroom som jag tor beror på att vi försöker be den planera en rutt för en taxi som redan är full, d.v.s. har capacity 0. Om du får samma fel hos dig så syns det tydligt i loggen när du kör denna branch. Jag har försökt filtrera bort sådana taxis här men får inte till det riktigt. Kan du förstå varför det blir så, är det ens en ny grej som uppstått nu? 🤔

Reply:
Vi ska lösa den buggen genom att istället för att skicka in capacity 0 så ska vi lägga upp alla planerade stop som den bilen har.

Reply:
dvs på truckToVehicle ska vi lägga in en property som heter steps som itererar igenom car.plan och sätter egenskaper enligt följande api:
https://github.com/VROOM-Project/vroom/blob/master/docs/API.md#steps

Reply:
Nu skickar jag in steps. Det har börjat skickas tillbaka ett annat fel från vroom, jag är osäker om det är relaterat eller inte. Försöker felsöka det.

```json
{
  "code": 2,
  "error": "Inconsistent amount length: 1 and 0."
}
```

Reply:
Ok men lös den här buggen genom att filtrera bort alla bilar som har 0 i capacity innan så kan vi skapa en egen PR för att hantera existerande last?